### ISSUE TITLE
Returning tuple variable results in IndexError

### ISSUE THREAD

**Issue by charles-cooper** [ethereum/vyper#1114]
### Version Information

* vyper Version: 8a0f9471b830762913513d69a1b3f9821106ee53
* OS: linux
* Python Version (python --version): Python 3.6.7
* Environment (output of `pip freeze`):
```
pycryptodome==3.7.2
vyper==0.1.0b5
```

### What's your issue about?
```python
### repro.vy
@private
def bug() -> (int128, address):
  ret: (int128, address)
  return ret
```
Throws an error:
```
  File "/home/charles/src-references/vyper/env/lib/python3.6/site-packages/vyper-0.1.0b5-py3.6.egg/vyper/parser/stmt.py", line 641, in parse_return
IndexError: list index out of range
```

### Fix ?
Prevent tuples as variables, as tuples could be easily misinterpreted and cause confusion when writing a smart contract.

**Comment by charles-cooper** [comment]
I have a potential fix at https://github.com/charles-cooper/vyper/tree/cmc-fix-tuple-return

**Comment by jacqueswww** [comment]
As discussed, I think we should prevent creating variables that are tuples ;)

**Comment by charles-cooper** [comment]
This seems resolved as of a7de2b21cc2582edeea9f67115431acec268fb5d. We could make another issue to prevent creating variables that are tuples.
